lib/docker-delta.coffee

```diff
@@ -10,7 +10,7 @@ btrfs = require './btrfs'
 utils = require './utils'
 Docker = require 'docker-toolbelt'
 
-docker = new Docker()
+docker = new Docker(socketPath: process.env.DOCKER_SOCKET ? '/var/run/docker.sock')
```

Comment:
@Page- `DOCKER_HOST` is the official env var the docker client and a lot of other docker related programs use. It can be either in the form of `unix:///var/lib/docker` or `tcp://example.com:1337`.

@pcarranzav you don't necessarily have to change it in the supervisor. The supervisor is a user of this library. You can set the right env at startup and then require the module. But I'd want the supervisor to at least understand both